src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -133,7 +130,7 @@ function get_item_html($itemkey) {
 			if (!$g['hideplatform']) {
 				$is_nano = ($g['platform'] == "nanobsd");
 				// there's extra info (boot slice) if it's nanobsd
-				$data_template = "%s" . ($is_nano ? "\n<br/>nanobsd: <strong>%s</strong>:<br/>\n%s\n%s\n%s" : '');
+				$data_template = "%s" . ($is_nano ? "\n<br/>nanobsd: <strong>%s</strong>:<br/>\n%s\n%s" : '');
```

Comment:
I think this has 1 too many %s - but I did not really test because actually on 2.4 there is not nanoBSD.